.github/workflows/cmake.yml

```diff
@@ -372,6 +378,21 @@ jobs:
       run: |
         brew install ninja ${{ matrix.packages }}
 
+    - name: Install Emscripten
+      if: contains(matrix.name, 'Emscripten')
+      run: |
+        git clone https://github.com/emscripten-core/emsdk.git
+        cd emsdk
+        ./emsdk install 1.39.6-upstream
+        ./emsdk activate 1.39.6-upstream
+        source ./emsdk_env.sh
+        echo "${PATH}" >> $GITHUB_PATH
```

Comment:
When adding whole path, it might be better not to append and instead replace, as it might add some paths twice and that might have unexpected results. We can assume that PATH equals to contents of file pointed by GITHUB_PATH at start of current CI step.

Reply:
Done (replaced with `echo "${PATH}" >> $GITHUB_PATH`).